imageio/plugins/pillow.py

```diff
@@ -286,6 +292,11 @@ class PNGFormat(PillowFormat):
         bits. In this case, given as a number between 1-256.
     dictionary (experimental): dict
         Set the ZLIB encoder dictionary.
+    interpret_uint8: bool
```

Comment:
I think renaming this to `prefer_uint8` would more accurately describe the behavior. What do you think?

Reply:
Perfect, the inaccuracy of "interpret" has been bugging me but I was stuck trying to come up with a better term.